### PR TITLE
python311Packages.cwl-upgrader: 1.2.8 -> 1.2.9, python311Packages.cwl-utils: 0.28 -> 0.29

### PR DESCRIPTION
**Pull request by fabaff**
## Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 253175` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>10 packages built:</summary>
  <ul>
    <li>cwltool</li>
    <li>cwltool.dist</li>
    <li>python310Packages.cwl-upgrader</li>
    <li>python310Packages.cwl-upgrader.dist</li>
    <li>python310Packages.cwl-utils</li>
    <li>python310Packages.cwl-utils.dist</li>
    <li>python311Packages.cwl-upgrader</li>
    <li>python311Packages.cwl-upgrader.dist</li>
    <li>python311Packages.cwl-utils</li>
    <li>python311Packages.cwl-utils.dist</li>
  </ul>
</details>

File: pkgs/development/python-modules/cwl-upgrader/default.nix [hunk 1]
```diff
@@ -11,7 +11,7 @@
 
 buildPythonPackage rec {
   pname = "cwl-upgrader";
-  version = "1.2.8";
+  version = "1.2.9";
   format = "setuptools";
 
   disabled = pythonOlder "3.7";
```
Comment (natsukium) [inline pkgs/development/python-modules/cwl-upgrader/default.nix:17]:
```suggestion
  disabled = pythonOlder "3.8";
```
https://github.com/common-workflow-language/cwl-upgrader/blob/6c50d8c5829dbdbad51e3b7691a0565fab7b2724/setup.py#L34

File: pkgs/development/python-modules/cwl-utils/default.nix [hunk 2]
```diff
@@ -16,7 +16,7 @@
 
 buildPythonPackage rec {
   pname = "cwl-utils";
-  version = "0.28";
+  version = "0.29";
   format = "setuptools";
 
   disabled = pythonOlder "3.7";
```
Comment (natsukium) [inline pkgs/development/python-modules/cwl-utils/default.nix:22]:
```suggestion
  disabled = pythonOlder "3.8";
```
https://github.com/common-workflow-language/cwl-utils/blob/c4d4b2f9ad1afbe1d9503375f66f59e835da97fb/setup.py#L26

File: pkgs/development/python-modules/cwl-upgrader/default.nix (hunk 1 above)
Comment (fabaff) [reply pkgs/development/python-modules/cwl-upgrader/default.nix]: Thanks, updated.

File: pkgs/development/python-modules/cwl-utils/default.nix (hunk 2 above)
Comment (fabaff) [reply pkgs/development/python-modules/cwl-utils/default.nix]: Thanks, updated.

**Pull request merged**